### PR TITLE
Added issue template for bug/issue report

### PR DESCRIPTION
**Pull request by grgprarup**
- Added issue template form to report issue or bugs

### CODE REVIEW

**Comment by koebel** [comment]
maybe also add template for PRs? 
see example at  https://github.com/owncloud/web/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by grgprarup** [comment]
> maybe also add template for PRs? see example at https://github.com/owncloud/web/blob/master/.github/PULL_REQUEST_TEMPLATE.md

The PR is on the way.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+name: 🐞 Bug Report
+description: File a bug report
+title: "[Bug]: <title>"
+labels: [ "bug" ]
+projects: [ "JankariTech/48" ]
```
Comment (saw-jan) [inline .github/ISSUE_TEMPLATE/bug.yaml:5]: what is this 48? :thinking:

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        examples:
+          - **OS**: Ubuntu 22.04
+          - **Node**: 18.17.1
+          - **pnpm**: 8.15.1
+          - **ownCloud Web**: ownCloud Web UI 9.0.0-alpha.1
+          - **oCIS**: Infinite Scale 5.1.0-prealpha+fa57da865 Community
```
Comment (saw-jan) [inline .github/ISSUE_TEMPLATE/bug.yaml:57]: if they are the placeholders
```suggestion
          - **ownCloud Web**: ownCloud Web UI 8.0.0
          - **oCIS**: Infinite Scale 5.0.0
```

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (grgprarup) [reply .github/ISSUE_TEMPLATE/bug.yaml:5]: That's the project number.

**Review by saw-jan — APPROVED** [review]

**Pull request merged**